src/Razor/src/Microsoft.AspNetCore.Razor.ProjectEngineHost/Serialization/MessagePack/SerializationFormat.cs

```diff
@@ -9,5 +9,5 @@ internal static class SerializationFormat
     // or any of the types that compose it changes. This includes: RazorConfiguration,
     // ProjectWorkspaceState, TagHelperDescriptor, and DocumentSnapshotHandle.
     // NOTE: If this version is changed, a coordinated insertion is required between Roslyn and Razor for the C# extension.
-    public const int Version = 4;
+    public const int Version = 5;
```

Comment:
//CC @DustinCampbell to check I updated the serialization stuff as expected.

Reply:
I think you'll also need to update `Microsoft.AspNetCore.Razor.Serialization.Json.SerializationFormat.cs`.